### ISSUE TITLE
DOMAIN IS EXPIRED

### ISSUE THREAD

**Issue by CallMeDimas** [Euro-pol/turnaround-api#11]
the domain https://turn.seized.live/ is expired, can u renew it... :)

**Comment by Euro-pol** [comment]
i might set it up on another domain later, i'll update the repo with it

**Comment by SNN4** [comment]
yes please . want to test on a check this on on cloudflare turnstile protected website .
